contents.json

```diff
@@ -5424,6 +5424,11 @@
       "category": "uicollectionview",
       "description": "An easy-to-use Collection View Layout for card-like animation.",
       "homepage": "https://github.com/netguru/CarLensCollectionViewLayout"
+    }, {
```

Comment:
@kingiol These changes shouldn't be here:
```
{
      "title": "DGElasticPullToRefresh",
      "category": "uitableview",
      "description": "Elastic pull to refresh.",
      "homepage": "https://github.com/gontovnik/DGElasticPullToRefresh"
}
```
Could you remove them from this commit or it would be event better if you make your changes from the `develop` branch?

Reply:
OK. Removed.